src/requests/__init__.py

```diff
@@ -157,7 +157,7 @@ def _check_cryptography(cryptography_version):
     __url__,
     __version__,
 )
-from .api import delete, get, head, options, patch, post, put, request
+from .api import delete, get, head, options, patch, post, put, request, send
```

Comment:
**Suggestion:** The import statement in __init__.py is importing 'send' which is not following the HTTP methods convention. Consider renaming it to a proper HTTP method or make it more generic if it's intended to handle multiple methods.
```suggestion
from .api import delete, get, head, options, patch, post, put, request, custom_request
```